src/src/Commands/GPIO.cpp

```diff
@@ -1190,28 +1190,28 @@ bool getGPIOPinStateValues(String& str) {
     // returns pin value using syntax: [plugin#xxxxxxx#pinstate#x]
     int par1;
     const bool validArgument = validIntFromString(gpio_descr, par1);
+    #if FEATURE_PINSTATE_EXTENDED
+    pluginID_t pluginID = INVALID_PLUGIN_ID;
+    #endif // if FEATURE_PINSTATE_EXTENDED
 
     if (validArgument) {
       switch (device[0]) {
         case 'g':
         {
-          const uint32_t key = createKey(PLUGIN_GPIO, par1);
-          const auto it = globalMapPortStatus.find(key);
-
-          if (it != globalMapPortStatus.end() && ((it->second.mode == PIN_MODE_PWM) || (it->second.mode == PIN_MODE_SERVO))) {
-            // For PWM or SERVO mode get the last set duty cycle
-            str = it->second.getValue();
-          } else {
-            // Just read the current pinstate
-            str = digitalRead(par1);
-          }
+          #if FEATURE_PINSTATE_EXTENDED
+          pluginID  = PLUGIN_GPIO;
+          #endif // if FEATURE_PINSTATE_EXTENDED
+          str       = digitalRead(par1);
```

Comment:
Doesn't this break existing features?

Reply:
No, it was all added for this feature, so it can be safely excluded if not needed/wanted.